docs/Getting_Started.md

```diff
@@ -1,71 +1,94 @@
-It's recommended to run the latest stable version that is at least 5.0 or higher.
-Beta versions may work, but expect limited support.
-#### Notable differences include
- - Supports more devices
- - Less built in libraries. If using RGB, bluetooth, and more, you will have to 
- add these libraries yourself
- - Some devices such as the nice!nano don't have much free space, so not all 
- features can be installed at the same time
- 
-## Getting KMK
-You can always find the latest releases on our CDN, in 
-[compiled and optimized](https://cdn.kmkfw.io/kmk-latest.zip) and 
-[raw, hackable text file](https://cdn.kmkfw.io/kmk-latest.unoptimized.zip) 
-forms. These follow the `master` branch here on GitHub. Just get the KMK folder
-and drop this directly in the CIRCUITPYTHON directory (not in a sub folder).
-Make sure to extract the zip, and put the `kmk` folder on the root of the
-CIRCUITPY drive on the microcontroller
-
-## Turning a controller into a keyboard
-### Supported keyboards
-If your keyboard and microcontroller are officially supported, simply visit the 
-page for your files, and dropping them on the root of the "flash drive". Those 
-pages can be found [here](https://github.com/KMKfw/boards). You will need the 
-`kb.py` and `main.py`. More advanced instructions can be found 
-[here](config_and_keymap.md). If using Curcuitpython and NOT KMKPython, you will
-also need [boot.py](https://github.com/KMKfw/kmk_firmware/blob/master/boot.py)
-
-### Porting a keyboard
-If you are porting a board to KMK, check the page [here](porting_to_kmk.md). 
-
-### Handwired Keyboard
-If you are doing a hand wire, check [here](handwiring.md)
-
-## Additional features
-Want to have fun features such as RGB, split keyboards and more? Check out what 
-extensions can do [here](extensions.md)
-
-## Debugging
-Debugging help can be found [here](debugging.md)
+> Life was like a box of chocolates. You never know what you're gonna get.
+
+KMK is a keyboard focused layer that sits on top of [CircuitPython](https://circuitpython.org/). As such, it should work with most [boards that support CircuitPython](https://circuitpython.org/downloads). It is best to use the last stable version (>5.0).
+Known working and recommended devices can be found [here](Officially_Supported_Microcontrollers.md)
+
+We are also providing a keyboard optimized version of CircuitPython (simplified to cope with memory limits of certain boards and with a selection of preinstalled relevant modules).If you're wondering why use KMKPython rather than barebone CircuitPython, we tried to compare both approaches [here](kmkpython_vs_circuitpython.md)
+
+<br>
+
+## TL;DR Quick start guide
+> To infinity and beyond!
+1. [Install CircuitPython on your board](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython). With certain boards (pico, I look at you !), it can be as easy as drag and dropping the firmware on the drive
```

Comment:
Most boards are a simple drag and drop, so maybe omit the comment as it's not at all pico specific here.